subsys/bluetooth/host/conn.c

```diff
@@ -53,6 +53,9 @@ struct tx_meta {
 	bool is_cont;
 };
 
+BUILD_ASSERT(sizeof(struct tx_meta) == CONFIG_BT_CONN_TX_USER_DATA_SIZE,
+	     "User data size is wrong!");
```

Comment:
I'm confused. Why do you create a modifiable Kconfig variable for this, if it's only allowed to have one single value (`sizeof(struct tx_data)`)?

Reply:
Nevermind. Just saw that it has no prompt, i.e. it's not modifiable by the app. Still not sure why a Kconfig option is needed.. so that you have something you can use as reference in net subsystem Kconfig files?

Reply:
> so that you have something you can use as reference in net subsystem Kconfig files?

Yes, that would be why, otherwise a simple `#define` would have been sufficient